### ISSUE TITLE
Reproduction and comparision to video-nonlocal-net

### ISSUE THREAD

**Issue by gurkirt** [facebookresearch/SlowFast#37]
Hi, Thank you for making this code available. 

I am trying to reproduce the result from your code. I got 72.6% top1 accuracy for I3D, from (Kinetics/I3D_8x8_R50.yaml)[https://github.com/facebookresearch/SlowFast/blob/master/configs/Kinetics/I3D_8x8_R50.yaml], on 8 GPUs without modifying any other parameter, and for 196 epochs.

I suppose that is from the fact that version of the kinetics dataset that I have is missing about 600 validation and 9K training videos. It would be a great help you please make some comment about this?

Another question about comparing the results of this repo with (video-nonlocal-net repo)[https://github.com/facebookresearch/video-nonlocal-net]. For instance, I3D_8x8_R50 have an accuracy of 73.5 here and 73.5 there but the initialisation here is random their (video-nonlocal-net repo) uses imagenet initialization. However, the number of epochs there is much less (~100 epochs). 

What makes random initialisation work as good as imagenet initialisation, as in video-nl-net repo? Is it just the number of epochs or something else?
How does one compare them?

Looking forward to your answers,
Gurkirt

**Comment by takatosp1** [comment]
Hi,

Thanks for using the codebase!
Regarding to the data, as mentioned in another thread, because of the legal issue, we are not allowed to release the video. Having less training or validation video will cause a performance drop in training or testing. 

if you want to *reproduce* the result of nonlocal network, you need to use the ImageNet pretrain and finetune it.
If you want to play the recipe of training from scratch proposed in slowfast paper, you shall use slowonly 8x8 recipe.

Thanks

**Comment by gurkirt** [comment]
Can you share the validation videos, offline?

**Comment by takatosp1** [comment]
I’m working on a legal review now.

**Comment by gurkirt** [comment]
Great, thanks!

**Comment by gurkirt** [comment]
Also, res50 imagenet pre-train weights are not available from https://dl.fbaipublicfiles.com/pyslowfast/model_zoo/R50_IN1K.pyth

**Comment by gurkirt** [comment]
@takatosp1 any news on the data?

**Comment by takatosp1** [comment]
Hi, you should find the pretrain weight is back online if you check the link now. I'll push the IN1K pretrain yaml config to the codebase in two days. (running a sanity check now) 
Let me know you need anything else.

**Comment by gurkirt** [comment]
I already got the date from Kinetics/ in ActivityNet, I get 71.54 from I3D weight weights provided, where top1 should be close to 73.5. Of course, I used the same preprocessing as in video-nonlocal-net.

I would go a long way to reproduce the result, if you could release the validation data at least if not training. 

Thanks
Gurkirt
